src/containers/Cfe/test/selectors.test.ts

```diff
@@ -55,6 +55,13 @@ describe("Viewer selectors", () => {
                 fovDownloadHref: "&id=F12762",
                 fovPath: "fovVolumeviewerPath",
                 viewerChannelSettings: {},
+                metadata: {
+                    "Anaphase segmentation complete": "undefined",
```

Comment:
This field is `"undefined"` due to a key mismatch in mocks.ts - the featureDef has the key `anaphase-segmentation-complete`, but the apparently matching entry in the data object is called `cell-segmentation`. Is this intentional? Changing the keys to match breaks other tests.